utilities/general-helpers.metta

```diff
@@ -4,8 +4,8 @@
 (= (~= $a $b) (not (== $a $b)))
 
 ;; INFO: Uncomment for the use of MeTTaLog.
-;; (: >= (-> Number Number Bool))
-;; (= (>= $a $b) (or (== $a $b) (> $a $b)))
+(: >= (-> Number Number Bool))
```

Comment:
Why did you uncomment this line? It was supposed to be for metta log definition only.

Reply:
This could result non deterministic evaluation of the built in `>=` function.